src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: string;
+  description: string;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={`${name} avg`}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={formatDelayFooterTitle(metric.meta.value as number[], metric.meta.type)}
+              description={metric.meta.description}
+            />
+          )
+        }
+      />
+    );
+  } else if (name === 'Load') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={name}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={metric.meta.title}
+              description={metric.meta.description}
+            />
+          )
+        }
+      />
+    );
+  } else if (name === 'Response time avg') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={name}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={formatNumber(metric.meta.value![0], metric.meta.type)}
```

Comment:
Same comment as above, could we get rid of the `!` by checking `metric.meta?.value.length &&` or similar?